bluebrain/deployment/environments/applications_libraries.yaml

```diff
@@ -25,6 +25,7 @@ spack:
         - py-scikit-learn
         - py-scipy
         - py-tensorflow
+        - py-websockets
```

Comment:
Both these modifications should disappear again, no changes to `libraries` needed.

I overlooked it, but `py-websockets` needs to be a dependency with `type=("build", "run")`, as that should be always done for Python dependencies.

Reply:
Thanks! I applied these changes, but no luck yet.